Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.cs

```diff
@@ -412,5 +412,36 @@ public void Reset()
         {
             InitImageLayout(true);
         }
+
+        /// <summary>
+        /// Tries to set a new value for the cropped region, returns true if it succeeded, false if the region is invalid
+        /// </summary>
+        /// <param name="rect">The new cropped region.</param>
+        /// <returns>bool</returns>
+        public bool TrySetCroppedRegion(Rect rect)
+        {
+            // Reject regions smaller than the minimum size
+            if (rect.Width < MinCropSize.Width || rect.Height < MinCropSize.Height)
+            {
+                return false;
+            }
+
+            // Reject regions larger than the original picture
+            if (rect.Width > _restrictedCropRect.Width || rect.Height > _restrictedCropRect.Height)
```

Comment:
Thanks, but don't forget to check `rect.X` and `rect.y`.

Reply:
Oops right didn't realize they can have different offsets, fixing this line

Reply:
@arcadiogarcia did this piece get resolved properly?  Is that the next section with the `Left` and `Bottom` checks?  Just wanted to make sure before approving.

Reply:
@michael-hawker Yes, that new condition should reject all regions that step outside the original picture.